src/components/SideColumnNavBar.tsx

```diff
@@ -0,0 +1,163 @@
+                data-cy={`side-column-nav-bar-categorical-${column.header}`}
+                key={columnId}
+                selected={selectedColumnId === columnId}
+                onClick={() => onSelectColumn(columnId, 'Categorical')}
+                sx={{
+                  '&:hover': {
+                    backgroundColor: 'action.hover',
+                  },
+                }}
+              >
+                <Typography>{column.header}</Typography>
+              </ListItemButton>
+            ))}
+          </List>
+        </Collapse>
+      </div>
+
+      <div data-cy="side-column-nav-bar-continuous">
+        <Button
+          data-cy="side-column-nav-bar-continuous-toggle-button"
+          className="justify-start"
+          fullWidth
+          onClick={() => toggleExpansion('continuous')}
+          endIcon={expanded.continuous ? <ArrowDropUpIcon /> : <ArrowDropDownIcon />}
+        >
+          <Typography
+            sx={{
+              fontWeight:
+                selectedColumnId && continuousColumns.some(([id]) => id === selectedColumnId)
+                  ? 'bold'
+                  : 'normal',
+            }}
+          >
+            Continuous
+          </Typography>
+        </Button>
+        <Collapse in={expanded.continuous}>
+          <List>
+            {continuousColumns.map(([columnId, column]) => (
+              <ListItemButton
+                data-cy={`side-column-nav-bar-continuous-${column.header}`}
+                key={columnId}
+                selected={selectedColumnId === columnId}
+                onClick={() => onSelectColumn(columnId, 'Continuous')}
+                sx={{
+                  '&:hover': {
+                    backgroundColor: 'action.hover',
+                  },
+                }}
+              >
+                <Typography>{column.header}</Typography>
+              </ListItemButton>
+            ))}
+          </List>
+        </Collapse>
+      </div>
+
+      <div data-cy="side-column-nav-bar-other">
+        <Button
+          data-cy="side-column-nav-bar-other-toggle-button"
+          className="justify-start"
+          fullWidth
+          onClick={() => toggleExpansion('other')}
+          endIcon={expanded.other ? <ArrowDropUpIcon /> : <ArrowDropDownIcon />}
+        >
+          <Typography
+            sx={{
+              fontWeight:
+                selectedColumnId && otherColumns.some(([id]) => id === selectedColumnId)
+                  ? 'bold'
+                  : 'normal',
+            }}
+          >
+            Other
+          </Typography>
+        </Button>
+        <Collapse in={expanded.other}>
+          <List>
+            {otherColumns.map(([columnId, column]) => (
+              <ListItemButton
+                data-cy={`side-column-nav-bar-other-${column.header}`}
+                key={columnId}
+                selected={selectedColumnId === columnId}
+                onClick={() => onSelectColumn(columnId, null)}
```

Comment:
I vaguely remember we didn't want these columns to be clickable / interactive, no? Or do you want to handle that behavioural difference elsewhere, e.g. with some notice of "can't annotate this column" that shows up when clicked here?

Reply:
I believe we decided to make the lowest level of `unannotated` type non-interactable which would be handled once we implement the logic for annotation.